### PR TITLE
Fix 'X' key shortcut with multiple scenes open

### PR DESCRIPTION
**Pull request by rsredsq**

### CODE REVIEW

File: Script/AtomicEditor/ui/Shortcuts.ts [hunk 1]
```diff
@@ -99,10 +99,13 @@ class Shortcuts extends Atomic.ScriptObject {
 
     }
 
-    invokeGizmoAxisModeChanged(mode: Editor.AxisMode, toggle: boolean = false) {
-
-        this.sendEvent("GizmoAxisModeChanged", { mode: mode, toggle: toggle });
+    toggleGizmoAxisMode() {
+        var editor = EditorUI.getMainFrame().resourceframe.currentResourceEditor;
 
+        if (editor instanceof Editor.SceneEditor3D) {
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/Shortcuts.ts]: Excellent, I think this is the first usage of instanceof :)

Comment (rsredsq) [inline Script/AtomicEditor/ui/Shortcuts.ts]: yeah, much better than `if (editor.typeName == "SceneEditor3D")` :D

File: Script/AtomicEditor/ui/Shortcuts.ts [hunk 2]
```diff
@@ -99,10 +99,13 @@ class Shortcuts extends Atomic.ScriptObject {
 
     }
 
-    invokeGizmoAxisModeChanged(mode: Editor.AxisMode, toggle: boolean = false) {
-
-        this.sendEvent("GizmoAxisModeChanged", { mode: mode, toggle: toggle });
+    toggleGizmoAxisMode() {
+        var editor = EditorUI.getMainFrame().resourceframe.currentResourceEditor;
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/Shortcuts.ts]: I wonder if we shouldn't have a more direct way of accessing the current resource editor?  Could be a follow up

File: Script/Packages/Editor/Editor.json [hunk 3]
```diff
@@ -6,5 +6,5 @@
 							 "Source/AtomicEditor/Editors", "Source/AtomicEditor/Editors/SceneEditor3D",
 						   "Source/AtomicEditor/Components"],
 	"classes" : ["EditorMode", "PlayerMode", "FileUtils", "ResourceEditor", "JSResourceEditor",
-							 "SceneEditor3D", "SceneView3D", "SceneSelection", "EditorComponent", "CubemapGenerator"]
+							 "SceneEditor3D", "SceneView3D", "SceneSelection", "EditorComponent", "CubemapGenerator", "Gizmo3D"]
 }
```
Comment (JoshEngebretson) [inline Script/Packages/Editor/Editor.json]: Make it scriptable, it is just that easy! :+1:

**Comment by JoshEngebretson** [comment]
Nice work, will land when passed CI

File: Script/AtomicEditor/ui/Shortcuts.ts (hunk 2 above)
Comment (rsredsq) [inline Script/AtomicEditor/ui/Shortcuts.ts]: exactly, we have that https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/ui/EditorUI.ts#L51 :sweat_smile:

Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/Shortcuts.ts]: Ok, let's use that instead

File: Script/AtomicEditor/ui/Shortcuts.ts [hunk 4]
```diff
@@ -46,7 +46,7 @@ class Shortcuts extends Atomic.ScriptObject {
 
     invokeFormatCode() {
 
-        var editor = EditorUI.getMainFrame().resourceframe.currentResourceEditor;
+        var editor = EditorUI.getCurrentResourceEditor();
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/Shortcuts.ts]: Nice, good cleanup, thanks

**Pull request merged**